packages/terra-profile-image/src/terra-dev-site/doc/example/common/FitTypesWrapper.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+function FitTypesWrapper(WrappedComponent) {
+  return class FitTypes extends React.Component {
+    constructor(props) {
+      super(props);
+      this.handleOnSelect = this.handleOnSelect.bind(this);
+      this.state = { fitType: 'cover' };
+    }
+
+    handleOnSelect(event) {
+      this.setState({ fitType: event.target.value });
+    }
+
+    render() {
+      return (
+        <div>
+          <WrappedComponent fit={this.state.fitType} {...this.props} />
+          <p><label htmlFor="fitType">Select a Fit Type:</label></p>
+          <select id="fitType" name="fitType" value={this.state.fitType} onChange={this.handleOnSelect}>
+            <option value="cover">cover</option>
+            <option value="contain">contain</option>
+            <option value="scale-down">scale-down</option>
+            <option value="none">none</option>
```

Comment:
Shouldn't there also be a test for the "fill" object fit type?

Reply:
Answered above: [PR 2804#discussion_r390656186](https://github.com/cerner/terra-core/pull/2804#discussion_r390656186)